cloud/azure/templates/azure_saml_ses/main.tf

```diff
@@ -22,7 +22,8 @@ module "app" {
   postgres_admin_login = var.postgres_admin_login
 
   # note that we must use GP tier
-  postgres_sku_name = "GP_Gen5_2"
+  postgres_sku_name   = "GP_Standard_D2s_v3"
+  postgres_storage_mb = "32768"
```

Comment:
**question:** Is this size a recommendation from somewhere? If so it would be helpful to add that as a comment.

Reply:
Related question: do we have any published recommendations on machine size for CiviForm? Is this equivalent to what we use for AWS?

Reply:
This is from https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-compute where the minimum storage size for general purpose flexible servers is 32GB. Since this is so much higher than the previous storage amount (5GB) I imagined it would suffice. 

but that does beg the question of what is the current minimum capacity of AWS storage?